### PR TITLE
PR for varBC sprint

### PR DESCRIPTION
**Pull request by BenjaminRuston**
jedi-ci-build-cache=rebuild

## Description

draft PR for variational bias correction sprint

companion to:
build-group=https://github.com/JCSDA-internal/ufo/pull/3015
build-group=https://github.com/JCSDA-internal/ufo-data/pull/368
build-group=https://github.com/JCSDA-internal/ioda/pull/1092

### CODE REVIEW

**Comment by PatNichols** [comment]
@BenjaminRuston Are you comfortable resolving the conflicts. The main thing to do is to go with yours but change the TOL_ZERO to TOL.

**Comment by PatNichols** [comment]
@BenjaminRuston comflicts resolved.

File: src/gsi_varbc/GsiAircraftBiasConverter.cc [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  Eigen::ArrayXXf biascoeffs(numIds, numPreds*3);
+
+  readObsBiasCoefficients(coeffile, biascoeffs);
+
+  for (int i = 0; i < numPreds; ++i) {
+    /// Set up the creation parameters for the bias coefficients variable
+    ioda::VariableCreationParameters float_params;
+    float_params.chunk = true;               // allow chunking
+    float_params.compressWithGZIP();         // compress using gzip
+    float missing_value = util::missingValue(missing_value);
```
Comment (PatNichols) [inline src/gsi_varbc/GsiAircraftBiasConverter.cc:83]:
```suggestion
    float missing_value = util::missingValue<float>();
```

**Review by fcvdb — APPROVED** [review]
Changes look good!

**Review by PatNichols — APPROVED** [review]

**Comment by PatNichols** [comment]
@BenjaminRuston noticed that your ufo build group was pointed at the wrong PR.

**Pull request merged**